code/scripting/api/libs/ui.cpp

```diff
@@ -542,6 +556,225 @@ ADE_FUNC(getCmdBriefing,
 	return ade_set_args(L, "o", l_CmdBrief.Set(Cmd_briefs[0]));
 }
 
+//**********SUBLIBRARY: UserInterface/Debriefing
+ADE_LIB_DERIV(l_UserInterface_Debrief,
+	"Debriefing",
+	nullptr,
+	"API for accessing data related to the debriefing UI.<br><b>Warning:</b> This is an internal "
+	"API for the new UI system. This should not be used by other code and may be removed in the future!",
+	l_UserInterface);
+
+ADE_FUNC(initDebriefing,
+	l_UserInterface_Debrief,
+	nullptr,
+	"Builds the debriefing, the stats, sets the next campaign mission, and makes all relevant data accessible",
+	"number",
+	"Returns 1 when completed")
```

Comment:
It would be better to return `true` when completed, unless there are other possible return values.  Alternatively, the return value may not be needed at all.